packages/xo-server/src/vhd-merge.js

```diff
@@ -235,26 +225,22 @@ export class Vhd {
   getEndOfHeaders () {
     const { header } = this
 
-    let end = uint32ToUint64(this.footer.dataOffset) + VHD_HEADER_SIZE
+    let end = this.footer.dataOffset + VHD_HEADER_SIZE
```

Comment:
This seems incorrect, what do you think?

Reply:
it seem to be computing the end of the header correctly to me.

Reply:
`footer.dataOffset` is always the offset of the `header`?

Reply:
It is always the address of the header. The next question is "are the footer and the header always next to each other without anything interspacing?", I suspect the answer is yes because of the spec: "This field holds the absolute byte offset, from the beginning of the file, to the next structure."

Reply:
Our code only support footer + header at position 512 and 1024, we should probably assume this fact completely and not use this entry, what do you think?

Reply:
agreed